lib/survey_gizmo/api/page.rb

```diff
@@ -9,20 +11,27 @@ class Page
     attribute :properties,    Hash
     attribute :after,         Integer
     attribute :survey_id,     Integer
+    attribute :questions,     Array[Question]
```

Comment:
this is the other big change - i noticed that SG is now handing down information about pages in `Survey` responses, and `Questions` within `Pages` and `Options` within questions, all as one API request for `Survey`, and Virtus can automagically parse this out into those types of objects.  (except for sub questions, which are still broken)

So now all the data about a survey's questions comes down with one API request instead of like tens or hundreds.

the great unknown here is whether the data returned this way is actually the same as data returned by an individual api request, because that is [not always the case](https://github.com/RipTheJacker/survey-gizmo-ruby/blob/c399d416a1797062d5cb7c053577635125e8b819/lib/survey_gizmo/api/survey.rb#L56-L57).  my somewhat limited testing said they are the same but i didn't try to confirm every single data point, just some basic spot checking.